include/deal.II/lac/petsc_parallel_block_vector.h

```diff
@@ -100,11 +100,27 @@ namespace PETScWrappers
                             const size_type     local_size);
 
       /**
-       * Copy-Constructor. Set all the properties of the parallel vector to
+       * Copy constructor. Set all the properties of the parallel vector to
        * those of the given argument and copy the elements.
        */
       BlockVector (const BlockVector  &V);
 
+#ifdef DEAL_II_WITH_CXX11
+      /**
+       * Move constructor. Creates a new vector by stealing the internal data
+       * of the vector @p v.
+       *
+       * @note This operator is only available if deal.II is configured with
+       * C++11 support.
+       */
+      BlockVector (BlockVector &&v)
+      {
+        swap(v);
+        // be nice and reset v to zero
+        v.reinit(0, v.get_mpi_communicator(), 0, 0, false);
```

Comment:
This last line may actually be expensive -- it requires communication. It seems like the right thing to do, but at the same time nobody's going to touch the resulting vector. Could we just omit the reinit?